std/assembly/util/string.ts

```diff
@@ -66,14 +481,121 @@ export const enum CharCode {
   z = 0x7A
 }
 
-// 23 * 8 = 184 bytes
 // @ts-ignore: decorator
-@lazy @inline
-const POWERS10: StaticArray<f64> = [
-  1e00, 1e01, 1e02, 1e03, 1e04, 1e05, 1e06, 1e07, 1e08, 1e09,
-  1e10, 1e11, 1e12, 1e13, 1e14, 1e15, 1e16, 1e17, 1e18, 1e19,
-  1e20, 1e21, 1e22
-];
+@inline
+export function isAscii(c: u32): bool {
+  return !(c & ~0x7F);
```

Comment:
Not sure if worth, but `!(c >>> 7)` maybe for a smaller constant?

Reply:
Good catch! This shaved 1 byte =)